sequencing/ATACSeq/jobSubmission/1_batchRunPreAnalysis.sh

```diff
@@ -0,0 +1,159 @@
+##                               ATAC-seq pipeline STEP 1: Pre-analysis                                               ##
+## ===================================================================================================================##
+## PROOFREADER : Marina Flores-Payan                                                                                  ||
+## CONTACT: m.flores-payan@exeter.ac.uk                                                                               ||
+## LAST UPDATE: February 2024                                                                                         ||
+## ===================================================================================================================##
+## EXECUTION: sbatch --array= ./sequencing/ATACSeq/jobSubmission/1_batchRunPreAnalysis.sh <projectName> <option>      ||
+## - execute from scripts directory                                                                                   ||
+##                                                                                                                    ||
+## INPUTS:                                                                                                            || 
+## --array -> Number of jobs to run. Will select sample(s) corresponding to the number(s) input                       ||
+## $1 -> <projectName> Name of project                                                                                ||
+## $2 -> <option> Specify step to run: FASTQC, TRIM, ALIGN or ENCODE. Can be combined. Default is to                  ||
+## run all                                                                                                            ||
+##                                                                                                                    ||
+## DESCRIPTION: This script performs the first step in the ATAC-seq pipeline: pre-analysis. This                      ||
+## includes quality control of raw reads (FASTQC), trimming the reads (TRIM), aligning reads to                       ||
+## reference genome (ALIGN) and calculate ENCODEQC metrics (ENCODE).                                                  ||
+##                                                                                                                    ||
+## REQUIRES:                                                                                                          ||
+## - File in ${METADIR}/samples.txt that lists sample names.                                                          ||
+## ===================================================================================================================##
+
+## ============ ##
+##    SET UP    ##
+## ============ ##
+
+## print start date and time when script runs
+echo Job started on:
+date -u
+
+## needs to be executed from the scripts folder
+echo "Changing Folder to: " $SLURM_SUBMIT_DIR
+cd $SLURM_SUBMIT_DIR || exit 1
```

Comment:
These commands are not doing anything, I don't know why they were in the original file.